### PR TITLE
[PAM][TR] PIM-4338: Display relevant thumbnails of assets

### PR DESCRIPTION
**Pull request by phaseinducer**
https://akeneo.atlassian.net/browse/PIM-4338

| Q                  | A
| ------------------ | ---
| Migration script?  | Y
| Behats             | Y
| Specs & Static     | Y
| Changelog updated  | Y

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+- Move `Pim\Bundle\DataGridBundle\Datagrid\Product\ConfigurationRegistry` to `Pim\Bundle\DataGridBundle\Datagrid\Configuration\Product\ConfigurationRegistry`
+- Move `Pim\Bundle\DataGridBundle\Datagrid\Product\ContextConfigurator` to `Pim\Bundle\DataGridBundle\Datagrid\Configuration\Product\ContextConfigurator`
+- Move `Pim\Bundle\DataGridBundle\Datagrid\Product\FiltersConfigurator` to `Pim\Bundle\DataGridBundle\Datagrid\Configuration\Product\FiltersConfigurator`
+- Move `Pim\Bundle\DataGridBundle\Datagrid\Product\GroupColumnsConfigurator` to `Pim\Bundle\DataGridBundle\Datagrid\Configuration\Product\GroupColumnsConfigurator`
+- Move `Pim\Bundle\DataGridBundle\Datagrid\Product\SortersConfigurator` to `Pim\Bundle\DataGridBundle\Datagrid\Configuration\Product\SortersConfigurator`
+- Move `Pim\Bundle\DataGridBundle\Datagrid\RequestParametersExtractor` to `Pim\Bundle\DataGridBundle\Datagrid\Request\RequestParametersExtractor`
+- Move `Pim\Bundle\DataGridBundle\Datagrid\RequestParametersExtractorInterface` to `Pim\Bundle\DataGridBundle\Datagrid\Request\RequestParametersExtractorInterface`
```
Comment (nidup) [inline CHANGELOG.md]: we can add sed commands in Upgrade.md partners often override these configurators in projects

File: src/Pim/Bundle/DataGridBundle/Resources/config/configurators.yml [hunk 2]
```diff
@@ -18,33 +17,57 @@ services:
...
+
+    # deprecated, will be removed in 1.5, please use pim_datagrid.datagrid.configuration.product.sorters_configurator
+    pim_datagrid.datagrid.product.sorters_configurator:
+        alias: pim_datagrid.datagrid.configuration.product.sorters_configurator
+
+    # deprecated, will be removed in 1.5, please use pim_datagrid.datagrid.configuration.product.group_columns_configurator
+    pim_datagrid.datagrid.product.group_columns_configurator:
+        alias: pim_datagrid.datagrid.configuration.product.group_columns_configurator
+
+    # deprecated, will be removed in 1.5, please use pim_datagrid.datagrid.request.parameters_extractor
```
Comment (nidup) [inline src/Pim/Bundle/DataGridBundle/Resources/config/configurators.yml]: :+1:

**Comment by nidup** [comment]
baby comment then GTM

File: CHANGELOG.md (hunk 1 above)
Comment (phaseinducer) [inline CHANGELOG.md]: done

**Comment by jjanvier** [comment]
GTM

**Pull request merged**